Sources/TuistGenerator/Generator/SchemeDescriptorsGenerator.swift

```diff
@@ -552,7 +552,7 @@ final class SchemeDescriptorsGenerator: SchemeDescriptorsGenerating {
             region: scheme.runAction?.options.region,
             launchAutomaticallySubstyle: launchActionConstants.launchAutomaticallySubstyle,
             storeKitConfigurationFileReference: storeKitConfigurationFileReference,
-            customLLDBInitFile: scheme.runAction?.customLLDBInitFile
+            customLLDBInitFile: scheme.runAction?.customLLDBInitFile?.pathString
```

Comment:
We'll need to convert back to relative here otherwise the generated project wouldn't be compatible on different machines if checked in

see store kit example above 

https://github.com/tuist/tuist/pull/5071/files#diff-1efcecec8991d63da041bac8df08171945a6a432a24add71615c13679e9ee836R483

Reply:
push commit for this thank you @kwridan